src/providers/data_providers/googlePlacesBusinessImageProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { BusinessInfoInput, Geocode } from '../../models/data_models/types';
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { GooglePlacesApiUtils } from '../../utils/googlePlacesApiUtils';
+import { Image, ImageProviderName } from '../../models/data_models/image';
+
+export class GooglePlacesBusinessImageProvider extends BaseBusinessImageProvider {
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		const googlePlacesApi = await new GooglePlacesApiUtils(this.config).init();
+		return googlePlacesApi
+			.getPlaceID(businessInfo.address, businessInfo.name)
+			.then((id) => googlePlacesApi.getPlaceDetails(id))
+			.then((photoRefs) =>
+				Promise.all(photoRefs.map((photoRef) => googlePlacesApi.getPhoto(photoRef)).filter((x) => x))
```

Comment:
Isn't this `.filter` running on a list of promises? I think that promises are always evaluated as `true`, so this might not be working as expected. It might make more sense to move the `.filter` to the next `.then`, so it can run before the `.map` on a list without promises.